CHANGELOG.md

```diff
@@ -20,25 +20,191 @@ We’ve updated the header component to support account information and links. A
 
 This was added in [pull request #1058: New header with account section](https://github.com/nhsuk/nhsuk-frontend/pull/1058).
 
+#### Create custom width container classes
+
+You can now create custom page width container classes using the `nhsuk-width-container` mixin. You do this by passing in the required maximum width of the container.
+
+For example:
+
+```scss
+.app-width-container--wide {
+  @include nhsuk-width-container(1200px);
+}
+```
+
+You can use the generated classes to set the width of:
+
+- template container
+- header container
+- footer container
+
+It was already possible to set the page app width with the `$nhsuk-page-width` variable. This new feature is useful when creating additional custom page width classes.
+
+This change was introduced in [#1412: Allow creating custom width containers](https://github.com/nhsuk/nhsuk-frontend/pull/1412)
+
 :boom: **Breaking changes**
 
 You must make the following changes when you migrate to this release, or your service might break.
 
+#### Update file paths
+
+To make sure NHS.UK frontend's files do not conflict with your code, we've moved our package files from `packages` to `dist/nhsuk`.
+
+##### If you’re using Sass
+
+Replace `packages/` with `dist/nhsuk` for any `@forward`, `@use` or `@import` paths in your [Sass](https://sass-lang.com/) files.
+
+Before:
+
+```scss
+@forward "node_modules/nhsuk-frontend/packages/core";
+```
+
+After:
+
+```scss
+@forward "nhsuk-frontend/dist/nhsuk/core";
+```
```

Comment:
Might be better to use the `@import` examples for now? Also maybe drop `/core` from the examples as I think it’s more common to import the whole thing?